CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
 # Change Log
+All notable changes to this project will be documented in this file.
```

Comment:
💡 Thanks for adding me to the review. 📓 Please note: all this is scaffolding text from the action, you can safely remove it and only keep `##` part. 

**Samples below:** Also some of the simple samples are here: (If you want to see sample and adapt that)

* https://github.com/Azure/vscode-aks-tools/blob/master/CHANGELOG.md 
* https://github.com/Azure/aks-periscope/blob/master/CHANGELOG.md

Reply:
thank you i've removed it